readme.md

```diff
@@ -160,43 +158,6 @@ You can programmatically access Favicons configuration (icon filenames, HTML, ma
 var config = require("favicons").config;
 ```
 
-### Gulp
-
-To use Favicons with Gulp, you can either use the [gulp-plugin](https://github.com/rejas/gulp-favicons) or call it manually as follows:
-
-```js
-var favicons = require("favicons").stream,
-  log = require("fancy-log");
-
-gulp.task("default", function () {
-  return gulp
-    .src("logo.png")
-    .pipe(
-      favicons({
-        appName: "My App",
-        appShortName: "App",
-        appDescription: "This is my application",
-        developerName: "Hayden Bleasel",
-        developerURL: "http://haydenbleasel.com/",
-        background: "#020307",
-        path: "favicons/",
-        url: "http://haydenbleasel.com/",
-        display: "standalone",
-        orientation: "portrait",
-        scope: "/",
-        start_url: "/?homescreen=1",
-        version: 1.0,
-        logging: false,
-        html: "index.html",
-        pipeHTML: true,
-        replace: true,
-      })
-    )
-    .on("error", log)
-    .pipe(gulp.dest("./"));
-});
-```
```

Comment:
Maybe we can provide alternative example how to use? But I agree, gulp is very rare today

Reply:
Such example would require importing `vinyl` package from `gulp`. And will not be as simple as current one.
This package will not be "gulp friendly" anymore. For those who need `gulp` there is still `gulp-plugin` available.

Reply:
Sounds good